RELEASE

```diff
@@ -55,61 +55,64 @@ cp -a \
     myocamlbuild.ml \
     Makefile \
     _tags \
-    $release
+    "${release}"
 
-mkdir $release/bin
+mkdir "${release}/bin"
 
 # Set the RELEASE to true by default.
 # sed -i 's/RELEASE *= *false/RELEASE = true/' $release
 
 echo 'Copying src ...'
 cp -a \
     src \
-    $release
+    "${release}"
 
 echo 'Copying tools ...'
 cp -a \
     tools \
-    $release
+    "${release}"
 
 echo 'Copying examples ...'
 cp -a \
     examples \
-    $release
+    "${release}"
 
-cd $release
-echo Deleting useless files:
-useless=$(find . -print | egrep '~|.DS_Store' || true)
-echo rm -rf $useless
-rm -rf $useless
+(
+    cd "${release}"
+    echo Deleting useless files:
+    useless=$(find . -print | grep -E '~|.DS_Store' || true)
+    echo rm -rf "${useless}"
+    rm -rf "${useless}"
 
-# Remove examples not yet ready for public consumption
-echo rm -rf examples/{notworking,cfg,beluga}
-rm -rf examples/{notworking,cfg,beluga}
+    # Remove examples not yet ready for public consumption
+    echo rm -rf examples/{notworking,cfg,beluga}
+    rm -rf examples/{notworking,cfg,beluga}
 
-echo "Copying $release to $release-sandbox ..."
-cd ..
-if [ -e $release-sandbox ]; then
+    echo "Copying ${release} to ${release-sandbox} ..."
+)
+
+if [ -e "${release}-sandbox" ]; then
     echo Release sandbox directory "$release-sandbox" already exists.
     exit 1
 fi
-cp -a $release $release-sandbox
+cp -a "${release}" "${release}-sandbox"
 
-cd $release-sandbox
+(
+    cd "${release}-sandbox"
 
-echo 'Building (bytecode) in release directory ...'
-make BYTE=1 || ( echo 'make failed; aborting release.' ; exit 1 )
+    echo 'Building (bytecode) in release directory ...'
+    make BYTE=1 || ( echo 'make failed; aborting release.' ; exit 1 )
```

Comment:
`BYTE=1` doesn't do anything anymore with our current Makefile, since it just invokes `dune build`. Have you tried running the RELEASE script to see if it still works?

Reply:
@tsani it kinda works (generate a tar file) with some errors, and I was not sure it is originally there or not...

Reply:
Well, if it generates a tar file, I guess there's no problem. We're not in any position to be making releases anyway, so we can cross this bridge when we come to it.

Reply:
@tsani I checked in `master`, and it works because I removed `-e`. I may have thought that this uses exit code much, so it should not use `-e`. I'm going to put `-e` again.

Reply:
Done. 055812727421a590bc0815ece82d207b264233bb